src/igvfd/schemas/phenotype_term.json

```diff
@@ -33,12 +33,12 @@
         },
         "term_id": {
             "@type": "@id",
-            "title": "Ontology ID",
-            "description": "An ontology identifier describing an assay.",
+            "title": "Term ID",
+            "description": "A term identifier describing an assay.",
```

Comment:
this is NOT an assay term, but phenotype? disease?

Reply:
done